mobile/src/test/java/org/openhab/habdroid/util/UtilTest.java

```diff
@@ -44,25 +37,26 @@ public void normalizeUrl() throws Exception {
 
     @Test
     public void parseOH1SitemapList() throws Exception {
-        List<OpenHABSitemap> sitemapList = Util.parseSitemapList(getSitemapOH1Document());
+        List<? extends Sitemap> sitemapList = Util.parseSitemapList(getSitemapOH1Document());
         assertFalse(sitemapList.isEmpty());
 
+        assertEquals(8, sitemapList.size());
+
         // Should be sorted
```

Comment:
So this isn't sorted anymore?

Reply:
it isn't, the XML and the JSON now are both unsorted to not confuse people when the XML is sorted in the app, but the JSON isn't :)